include/cantera/kinetics/StoichManager.h

```diff
@@ -650,11 +594,30 @@ class StoichManagerN
         _decrementReactions(m_cn_list.begin(), m_cn_list.end(), input, output);
     }
 
+    //! Return matrix containing stoichiometric coefficients
+    const Eigen::SparseMatrix<double>& stoichCoeffs() const
+    {
+        if (!m_finalized) {
+            // This can happen if a user overrides default behavior:
+            // Kinetics::finalizeSetup is not called after adding reactions via
+            // Kinetics::addReaction with the 'finalize' flag set to 'false'
+            throw CanteraError("StoichManagerN::stoichCoeffs",
+                "The object is not fully configured; make sure to call finalizeSetup.");
+        }
+        return m_stoichCoeffs;
+    }
+
 private:
+    bool m_finalized; //!< Boolean flag indicating whether setup is finalized
+
     std::vector<C1> m_c1_list;
     std::vector<C2> m_c2_list;
     std::vector<C3> m_c3_list;
     std::vector<C_AnyN> m_cn_list;
+
+    //! Sparse matrices for stoichiometric coefficients
+    std::vector<Eigen::Triplet<double>> m_coeffList;
```

Comment:
```suggestion
    SparseTriplets m_coeffList;
```
(or eliminate the typedef in `eigen_sparse.h` as it's otherwise unused)